packages/services/api/src/modules/schema/resolvers/Query/latestValidVersion.ts

```diff
@@ -9,24 +8,23 @@ export const latestValidVersion: NonNullable<QueryResolvers['latestValidVersion'
   args,
   { injector, session },
 ) => {
-  const { targetId, projectId, organizationId } = await injector
-    .get(IdTranslator)
-    .resolveTargetReference({
-      reference: args.target ?? null,
-      onError() {
-        throw new InsufficientPermissionError('project:describe');
-      },
-    });
+  const selector = await injector.get(IdTranslator).resolveTargetReference({
+    reference: args.target ?? null,
+  });
+
+  if (!selector) {
+    session.raise('project:describe');
+  }
```

Comment:
For some reason this does not make `selector` non-null below, causing a TS error...

The same code works fine in other parts...